docs/modules/ROOT/pages/cops_lint.adoc

```diff
@@ -2911,41 +2911,43 @@ require 'unloaded_feature'
 | Enabled by default | Safe | Supports autocorrection | VersionAdded | VersionChanged
 
 | Pending
-| No
-| Yes (Unsafe)
+| Yes
+| Yes
 | 0.93
 | -
 |===
 
 This cop checks for redundant safe navigation calls.
 
 In the example below, the safe navigation operator (`&.`) is unnecessary
-because `NilClass` has methods like `respond_to?` and `dup`.
-
-This cop is marked as unsafe, because auto-correction can change the
-return type of the expression. An offending expression that previously
-could return `nil` will be auto-corrected to never return `nil`.
+because `NilClass` has methods like `respond_to?` and `is_a?`.
```

Comment:
Unsafe examples include: `var = :to_a; foo&.respond_to?(var)` and expressions with `||`/`&&` returning `false` instead of `nil`.